src/test/java/com/github/tomakehurst/wiremock/HttpsBrowserProxyAcceptanceTest.java

```diff
@@ -344,24 +344,20 @@ public void certificatesSignedWithGeneratedRootCertificate() throws Exception {
   public void failsIfCaKeystorePathIsNotAKeystore() throws IOException {
     assertThrows(
         IOException.class,
-        () -> {
-          new WireMockServer(
-                  options()
-                      .enableBrowserProxying(true)
-                      .caKeystorePath(Files.createTempFile("notakeystore", "jks").toString()))
-              .start();
-        });
+        new WireMockServer(
+                options()
+                    .enableBrowserProxying(true)
+                    .caKeystorePath(Files.createTempFile("notakeystore", "jks").toString()))
+            ::start);
```

Comment:
This now fails early in CI, because it's not `start` that's throwing an error here, it's loading the keystore.
```
HttpsBrowserProxyAcceptanceTest > failsIfCaKeystorePathIsNotAKeystore() FAILED
    java.io.IOException: Short read of DER length
        at java.base/sun.security.util.DerInputStream.getLength(DerInputStream.java:571)
        at java.base/sun.security.util.DerValue.init(DerValue.java:411)
        at java.base/sun.security.util.DerValue.<init>(DerValue.java:352)
        at java.base/sun.security.util.DerValue.<init>(DerValue.java:365)
        at java.base/sun.security.pkcs12.PKCS12KeyStore.engineLoad(PKCS12KeyStore.java:1[97](https://github.com/wiremock/wiremock/actions/runs/4524677353/jobs/7968666662#step:9:98)3)
        at java.base/sun.security.util.KeyStoreDelegator.engineLoad(KeyStoreDelegator.java:222)
        at java.base/java.security.KeyStore.load(KeyStore.java:1479)
        at com.github.tomakehurst.wiremock.common.ssl.KeyStoreSource.load(KeyStoreSource.java:42)
        at com.github.tomakehurst.wiremock.common.ssl.KeyStoreSettings.loadStore(KeyStoreSettings.java:55)
        at com.github.tomakehurst.wiremock.jetty11.SslContexts.toX509KeyStore(SslContexts.java:113)
        at com.github.tomakehurst.wiremock.jetty11.SslContexts.buildSslContextFactory(SslContexts.java:71)
        at com.github.tomakehurst.wiremock.jetty11.SslContexts.buildManInTheMiddleSslContextFactory(SslContexts.java:52)
        at com.github.tomakehurst.wiremock.jetty11.Jetty11HttpServer.applyAdditionalServerConfiguration(Jetty11HttpServer.java:123)
        at com.github.tomakehurst.wiremock.jetty.JettyHttpServer.<init>(JettyHttpServer.java:[109](https://github.com/wiremock/wiremock/actions/runs/4524677353/jobs/7968666662#step:9:110))
        at com.github.tomakehurst.wiremock.jetty11.Jetty11HttpServer.<init>(Jetty11HttpServer.java:44)
        at com.github.tomakehurst.wiremock.jetty.JettyHttpServerFactory.buildHttpServer(JettyHttpServerFactory.java:31)
        at com.github.tomakehurst.wiremock.WireMockServer.<init>(WireMockServer.java:75)
        at com.github.tomakehurst.wiremock.HttpsBrowserProxyAcceptanceTest.failsIfCaKeystorePathIsNotAKeystore(HttpsBrowserProxyAcceptanceTest.java:350)
```